### ISSUE TITLE
Layer Effects

### ISSUE THREAD

**Issue by kennyjeffris** [1kc/razer-macos#98]
One of the best parts about Synapse is the ability to layer effects.  I see that being overkill for these purposes, but it would be great to be able to layer say spectrum and reactive, so that there is always a base color with reactive on top.  Great work, thanks for everything you do!

**Comment by gratefulsorceror** [comment]
Yes. This. Also those fancy effects like rainbow spin and fire would be nice too. Then again, this is pretty much shareware made by a man who isn't even getting paid for his work. lmao

**Comment by 1kc** [comment]
Yes, it would really be nice. Most of the effects in this app is pre-loaded in the hardware, so this app essentially just sets the keyboard into the right mode. There is actually a lot more engineering work involved if we want the cool effects done by programing individual keys. 😓

**Comment by cypherponk** [comment]
Seconding this (or fourth-ing?) Would be amazing to have a static base and reactive overlay, etc. Regardless, this app has changed my life--so thank you!

**Comment by hannojg** [comment]
Hey @1kc. Just talking about _layering_ effects. Would that be easily possible with the existing C-lib? I could invest some time to write a proper UI with which we can manage layer effects 😊
